client/my-sites/upgrades/cart/cart-plan-ad.jsx

```diff
@@ -32,12 +33,15 @@ const CartPlanAd = React.createClass( {
 		}
 
 		return (
-			<div className="cart-plan-ad">{
-				this.translate( 'Get this domain for free when you upgrade to {{strong}}WordPress.com Premium{{/strong}}!', {
-					components: { strong: <strong /> }
-				} )
-			} <a href="" onClick={ this.addToCartAndRedirect }>{ this.translate( 'Upgrade Now' ) }</a>
-			</div>
+			<CartAd>
+				{
+					this.translate( 'Get this domain for free when you upgrade to {{strong}}WordPress.com Premium{{/strong}}!', {
```

Comment:
Is it a good idea to move this product name string to a variable? We don't want translators to translate it, and it would make it easier to update the plan names, should we decide to. This should be a different PR since we do this in other places already...